e2e/src/page-objects/login.po.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  browser,
+  element,
+  by,
+  ElementFinder,
+  promise,
+  ElementArrayFinder
+} from "protractor";
+
+export class LoginPage {
+  navigateTo(): promise.Promise<any> {
+    return browser.get("/#/login");
+  }
+  async navigateToCourses(): promise.Promise<any> {
```

Comment:
if you use async await , no need to use promise.Promise

Reply:
changed